docs/blog/2022-january-community-update.rst

```diff
@@ -0,0 +1,124 @@
+.. post:: Jan 13, 2022
+   :tags: stats, year in review, newsletter
+   :author: Eric Holscher
+
+Write the Docs Community Kickoff - January 2022
+===============================================
+
+Hey folks, Eric Holscher here, one of the co-founders of Write the Docs.
+This is the January edition of the newsletter,
+which has a community update to give the newsletter team the month off.
+We'll keep it a bit shorter this year,
+but did want to update you on the community overall.
+
+New hybrid conference format
+----------------------
+
+Thanks to everyone who attended our virtual events in 2021.
+We had record attendance in Portland and Prague this year,
+and took a well-deserved break in Australia to let the community there have a rest.
+We continue to appreciate the warmth and spirit that you bring to our virtual events,
+and look forward to more of it in 2022.
+
+Our goal this year is to tentatively, and safely, come back to the in-person realm with our conferences.
+We're working to do this without losing the accessibility of the virtual events.
+Managing in-person and virtual is going to be a learning experience,
+but we think it will be worth it.
```

Comment:
```suggestion
Our goal this year is to tentatively, and safely, bring our conferences back to 
the in-person realm while keeping the improved accessibility provided by the
virtual events.
Managing a hybrid in-person *and* virtual conferences is going to be a huge 
learning experience, but we think it will be worth it.
```

Reply:
"Managing a hybrid" -> "Managing hybrid"